### PR TITLE
Add disable aggregator setting

### PR DESCRIPTION
**Pull request by vrtnd**

### CODE REVIEW

File: src/components/Aggregator/index.tsx [hunk 1]
```diff
@@ -1144,9 +1157,13 @@ export function AggregatorContainer({ tokenlist }) {
 							: null}
 					</span>
 
-					{isLoading && debouncedAmount && finalSelectedFromToken && finalSelectedToToken ? (
+					{isLoading && debouncedAmount && finalSelectedFromToken && finalSelectedToToken && enabledAdapters.length ? (
```
Comment (0xngmi) [inline src/components/Aggregator/index.tsx:1160]: why add this here?

File: src/components/Aggregator/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	const { isOpen, onClose } = useDisclosure({ defaultIsOpen: true });
+	const onCloseClick = () => {
+		onExternalClose();
+		onClose();
+	};
+	const onClick = (name) => (e) => {
+		const isChecked = e.target.checked;
+
+		setEnabledAdapters((adapters) =>
+			isChecked ? adapters.concat(name) : adapters.filter((adapterName) => adapterName !== name)
```
Comment (0xngmi) [inline src/components/Aggregator/Settings.tsx:29]: please don't shadow existing variables, adapters is already defined in the outer scope

**Review by 0xngmi — CHANGES_REQUESTED** [review]
biggest issue is that when we add new adapters those will be disabled by default to all our users

solution is to use a blacklist instead of a whitelist

File: src/components/Aggregator/Settings.tsx [hunk 3]
```diff
@@ -25,8 +25,8 @@ export const Settings = ({ adapters, enabledAdapters, setEnabledAdapters, onClos
 	const onClick = (name) => (e) => {
 		const isChecked = e.target.checked;
```
Comment (0xngmi) [inline src/components/Aggregator/Settings.tsx:26]: does this hold the state before the change or after the change on click?

Comment (vrtnd) [reply src/components/Aggregator/Settings.tsx:26]: event has new state after click

**Review by 0xngmi — APPROVED** [review]

**Review by 0xngmi — APPROVED** [review]

**Pull request merged**